<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inicio</class>
 <widget class="QDialog" name="inicio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alien Invasion: Last Hope</string>
  </property>
  <widget class="QPushButton" name="btnRegistrar">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Registrar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnIngresar">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>220</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ingresar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineUser">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="linePassword">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>260</width>
     <height>76</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Background-image: url(:/mapa/Alien Invasion_Last Hope.png)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNuevaPartida">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>123</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nueva Partida</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCargarPartida">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>123</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cargar Partida</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
